django/test/selenium.py

```diff
@@ -134,3 +139,17 @@ def disable_implicit_wait(self):
             yield
         finally:
             self.selenium.implicitly_wait(self.implicit_wait)
+
+    def is_page_accessible(self):
+        # id is used as an identifier here to accessibility test different states of
+        # the same URL, or we can restrict having only one is_page_accessible call on
+        # each unique URL
+        id = self.counter.get(self.selenium.current_url, 0)
+        self.counter[self.selenium.current_url] = id + 1
+
+        command = ["npx", "achecker", self.selenium.current_url + "#" + str(id)]
+        output = subprocess.run(command, capture_output=True, text=True)
```

Comment:
At the minute there isn't a strong link between the test and the output created from the test based off naming (if I wanted to update the baseline it would feel random). As tests can share urls then with the counter, the order you run tests also matter. Can we have the test name in the filepath?
I also don't love the naming convention with localhost etc - is this configurable at all? Or does it have to be generated from the url?

Reply:
>  Can we have the test name in the filepath?

No, this is not possible (or if it is, I was not able to find any docs on it). Basically the `baseline` folder is autogenerated and the folder is structured according to the path of the `URL` on which `achecker` is run. 

Another way to omit localhost is to add a hostname like `django-selenium-test` in the `/etc/hosts` file on the local system (on Linux, not sure about Windows or Mac) and point it to `127.0.0.1` similar to how `localhost` points to `127.0.0.1` - but this will make it difficult for a lot of other things.

Reply:
There is a boilerplate folder in the tool's repo - https://github.com/IBMa/equal-access/tree/master/accessibility-checker/boilerplates - that have some examples on baselines and other features. 

I found this https://github.com/IBMa/equal-access/tree/master/accessibility-checker/src#apis `getCompliance` which could be useful in naming baselines. I will take a look and come back.

Reply:
Nice :+1: you might also be able to write the html from the page into a tmp file with a well named filename/path and run the `achecker` against it (which might dictate the name of the report). Not sure if that will work though